src/BuildServer/BuildServerResponse.cs

```diff
@@ -0,0 +1,96 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    enum BuildState
```

Comment:
Is 'BuildState' being used or not? I couldn't find a place where it's used. If not, should we remove it

Reply:
This file is actually not used. I'm removing this file. Thanks Stella.